<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>408</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Nirmala UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: LightSkyBlue;</string>
  </property>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: green;
color: red;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;‡¶∞‡¶ø‡¶ï‡¶∂‡¶æ‡¶∞ ‡¶¢‡¶æ‡¶ï‡¶æ-‡ßß‡ß©&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
   <property name="text">
    <string>‡¶ú‡¶®‡ßç‡¶Æ ‡¶§‡¶æ‡¶∞‡¶ø‡¶ñ</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üîç ‡¶∏‡¶æ‡¶∞‡ßç‡¶ö</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
    </font>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>‡ß®‡ßÆ </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß®‡ßØ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß©‡ß¶</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß©‡ßß</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß©‡ß®</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß©‡ß©</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>‡ß©‡ß™ </string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
     </font>
    </property>
    <property name="text">
     <string>‡¶ì‡ßü‡¶æ‡¶∞‡ßç‡¶° ‡¶®‡¶æ‡¶Æ‡ßç‡¶¨‡¶æ‡¶∞</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
